javascript/packages/cli/package.json

```
@@ -50,7 +50,7 @@
     "url": "https://github.com/paritytech/zombienet/issues"
   },
   "dependencies": {
-    "@parity/zombienet-dsl-parser-wrapper": "^0.1.6",
+    "@zombienet/zombienet-dsl-parser-wrapper": "^0.1.6",
```

I think this should be:
```suggestion
    "@zombienet/dsl-parser-wrapper": "^0.1.6",
```

---

Yes, typo already fixed.